<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsAirport</class>
 <widget class="QDialog" name="StatisticsAirport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tb_statisticYear">
      <attribute name="title">
       <string>Статистика за год</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="wd_histogram" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_statisticMonth">
      <attribute name="title">
       <string>Статистика за месяц</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="wd_graphic" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="cb_month">
        <item>
         <property name="text">
          <string>Январь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Февраль</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Март</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Апрель</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Май</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Июнь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Июль</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Август</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сентябрь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Октябрь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ноябрь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Декабрь</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_closeStatusticAirport">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
